pkgs/development/python-modules/dnf4/default.nix

```
@@ -10,12 +10,16 @@
 , sphinx
 }:
 
+let
+  pyMajor = lib.head (lib.splitString [ "." ] python.version);
+in
+
 buildPythonPackage rec {
-  pname = "dnf4";
+  pname = "dnf4-unwrapped";
```

```suggestion
  pname = "dnf4";
```
we should probably rather change the wrappers name

---

I've changed this so that now the wrapper is called `dnf4` in all-packages.nix, and the unwrapped version is `dnf4` in python3Packages. Not sure if this is what you suggested.